src/view/tela_clientes.py

```diff
@@ -2,6 +2,8 @@
 
 from src.model.cliente import Cliente
 from src.utils.enum_tipo_cadastro import TipoCadastro
+from src.utils.codigo_gerador import GeradorCodigo
+from titi.ClothingStore.src.utils.validador import Validador
```

Comment:
Acho que esse import ta errado
```suggestion
from src.utils.validador import Validador
```